neusomatic/python/generate_dataset.py

```diff
@@ -568,8 +568,11 @@ def prepare_info_matrices_tabix(ref_file, tumor_count_bed, normal_count_bed, rec
             tag_normal_count_matrices, center, rlen, col_pos_map]
 
 
-def prep_data_single_tabix((ref_file, tumor_count_bed, normal_count_bed, record, vartype, rlen, rcenter, ch_order,
-                            matrix_base_pad, matrix_width, min_ev_frac_per_col, min_cov, ann, chrom_lengths)):
+def prep_data_single_tabix(input_record):
+
+    ref_file, tumor_count_bed, normal_count_bed, record, vartype, rlen, rcenter, ch_order, \
```

Comment:
Why do you need to unpack separately instead of in the argument list? Is this a Python3 thing?

Reply:
@marghoob That is because Python 3 does not accept def ((a,b,c)). For functions I call through multiprocess, I have one single input as a list.